src/validations/constraints/unit-tests/control-implementation-status-PASS.yaml

```
@@ -2,7 +2,7 @@
 test-case:
   name: The valid control implementation status test.
   description: Test that the specified control implementation status is valid.
-  content: ../content/ssp-all-VALID.xml
+  content: ../content/ssp-control-implementation-status-VALID.xml
```

Was this intentional and not meant to change like the others?

```suggestion
  content: ../../../content/rev5/examples/ssp/xml/fedramp-ssp-example.oscal.xml
```

Context: https://github.com/wandmagic/fedramp-automation/pull/18#pullrequestreview-2480097652

---

this was intentional. the example SSP doesn't leverage the implementation status prop, and in the nature of preserving it's integrity, i kept that valid check in isolated content.

---

OK I was getting errors on my PR so I guess I will have to sort that as well.